pgxnclient/commands/help.py

```diff
@@ -61,9 +61,12 @@ def find_all_commands(self):
         for p in path:
             if not os.path.isdir(p):
                 continue
-            for fn in os.listdir(p):
-                if fn.startswith('pgxn-'):
-                    rv.append(fn[5:])
+            try:
+                for fn in os.listdir(p):
+                    if fn.startswith('pgxn-'):
+                        rv.append(fn[5:])
+            except OSError as e:
+                return rv
```

Comment:
This means if any directory gives an error the following ones are not explored. That's too harsh